tasks.py

```diff
@@ -0,0 +1,83 @@
+from robocorp.tasks import task
+from robocorp import browser, http, excel
+from RPA.PDF import PDF
+
+@task
+def robot_spare_bin_python():
+    """Robot to enter weekly sales data into the RobotSpareBin Industries Intranet."""
+
+    # Without this, nothing is visible as it is running headless
+    browser.configure(
+        slowmo=100,
+        browser_engine="chromium",
+        headless=False,
+    )
+    open_the_intranet_website()
+    log_in()
+    download_excel_file()
+    fill_form_with_excel_data()
+    screenshot_results()
+    export_as_pdf()
+    log_out()
+
+
+def open_the_intranet_website():
+    """Navigates to the given URL"""
+
+    browser.goto("https://robotsparebinindustries.com/")
+
```

Comment:
...on the other hand, automatic formatters would add two empty lines between function definitions

Reply:
Apparently, for automatic formatters I need to install python locally (and the formatter), which doesn't go really well with "you do not even need to install python locally to write your robots"